app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -13,6 +13,7 @@ model: ProvisioningTemplate
 <%= ", Organization: [#{@organization.name}]" if @organization -%>
```

Comment:
\n added to the end of the string on every line would make this more readable, I think it used printing on new lines before you added -%> in other PR. Applicable to this and 3 lines below.

E.g. `"Organization: [#{@organization.name}]\n"`

we could drop `, ` in the beginning then